### ISSUE TITLE
Support calling kfuncs

### ISSUE THREAD

**Issue by jfernandez** [bpftrace/bpftrace#3169]
### Is your feature request related to a problem? Please describe.

There are some instrumentation use-cases where calling a [kfunc](https://docs.kernel.org/bpf/kfuncs.html) would be useful. For example, when you want to get the cgroup struct from an ID using `bpf_cgroup_from_id`, or when you want to grab a `task_struct` when only a pid is available, via `bpf_task_from_pid`.

Today, I need to write my own BPF program to achieve this type of instrumentation.

### Describe the solution you'd like

In BPF, you need to explicitly define which kfunc's your program will call, ie:
```
extern struct task_struct *bpf_task_from_pid(s32 pid);
```
bpftrace should also have something similar.

### Describe alternative solutions or features you've considered

Writing my own bpf program.

**Comment by viktormalik** [comment]
I have it on the top of my TODO list when I return from vacation.

**Comment by viktormalik** [comment]
The first instance of calling kfuncs is proposed in #3523. Once it is merged, I'd suggest to close this issue and open a separate issue for each kfunc that we'd like to support (`bpf_cgroup_from_id` and `bpf_task_from_pid` from this issue).

**Comment by danobi** [comment]
Done, closing like suggested by Viktor
